math/qalculate-qt/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           qmake5 1.0
+
+github.setup        qalculate qalculate-qt 3.20.1 v
+github.tarball_from releases
+revision            0
+
+categories          math
+platforms           darwin
+license             GPL-2+
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         Qalculate! is a multi-purpose desktop calculator
+long_description    {*}${description} \
+    It is small and simple to use but with much power and versatility \
+    underneath. Features include customizable functions, units, arbitrary \
+    precision, plotting, and a user-friendly interface.
+
+checksums           rmd160 a6030704584f0fb3e709ab928bcf3ad869ac2799 \
+                    sha256 189e952b2b6b99ef63db46bd70d31f140adf19e8364d0eb4064daefca6a16e8a \
+                    size 852543
+
+depends_lib-append  \
+                    port:libqalculate \
+                    port:qt5-qtbase
```

Comment:
The qmake5 PortGroup automatically includes this dependency.

Reply:
Thanks